odoo/addons/base/models/assetsbundle.py

```diff
@@ -229,7 +235,13 @@ def get_attachments(self, type, ignore_version=False):
         by file name and only return the one with the max id for each group.
         """
         version = "%" if ignore_version else self.version
-        url_pattern = '/web/content/%-{0}/{1}{2}.{3}'.format(version, self.name, '.%' if type == 'css' else '', type)
+        url_pattern = '/web/content/%-{0}/{1}{2}{3}.{4}'.format(
+            version,
+            self.name,
+            ('.%s' % self.user_direction if type == 'css' else ''),
```

Comment:
Antony does not want ltr assets suffixed with ltr

Reply:
@ged-odoo I was expecting this :) Will fix it soon

Reply:
OK, in this case I need to put ```.rtl``` somewhere else maybe before self.name otherwise ltr user served with rtl assets as url will be generated for ltr like /web/content/%-checksum/web.assets_backend.%.css